package.json

```diff
@@ -48,7 +48,6 @@
     "ember-data": "^2.8.0",
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
-    "ember-getowner-polyfill": "1.0.1",
```

Comment:
The dep here is still needed if we are intending to support Ember versions prior to 2.3, but we should update the version to `^1.1.0`.

The deprecation that is being fixed is specifically about refactoring `import getOwner from 'ember-getowner-polyfill';` to just `const { getOwner } = Ember;` (e.g. just use `Ember.getOwner`).